app/services/ipam/base.rb

```diff
@@ -41,6 +41,18 @@ def serializable_hash(options = nil)
 
     private
 
+    def block_ip_cache_key(ip)
```

Comment:
Naming wise: I wouldn't say this is blocked. Perhaps pre-reserved?

Reply:
The issue already occurs, if you press create-host multiple times. If you open the create host page 3 times you would currently get 3 times the same IP. 

well, from the user perspective the IP is blocked for its usage.

Reply:
I still think that's different from blocking it. It feels like a reservation or a lease. However, I won't push on that.

Reply:
opinion @lzap? Its easy to change if you have a proposal? Should we use "reserve"?

Reply:
Reserve feels a bit better but I do not mind blocking as well. Other synonyms: book, hold, order.